workspace/packages.dhall

```diff
@@ -0,0 +1,32 @@
+let upstream =
+      https://github.com/purescript/package-sets/releases/download/psc-0.15.4-20220901/packages.dhall
+        sha256:f1531b29c21ac437ffe5666c1b6cc76f0a9c29d3c9d107ff047aa2567744994f
+
+let additions =
+      { spec-reporter-codewars =
+        { repo = "https://github.com/andys8/purescript-spec-reporter-codewars"
+        , version = "e40d7b623fd52b336fb3007b9fefa8644ed45a55"
+        , dependencies =
+          [ "aff"
+          , "bigints"
+          , "console"
+          , "datetime"
+          , "debug"
+          , "effect"
+          , "exceptions"
+          , "foldable-traversable"
+          , "maybe"
+          , "prelude"
+          , "profunctor-lenses"
+          , "quickcheck"
+          , "rationals"
+          , "spec"
+          , "spec-discovery"
+          , "spec-quickcheck"
+          , "strings"
+          , "transformers"
```

Comment:
❓ All packages the user can import need to be added here, right?

I think some very common packages could be helpful: either, enums, record, tuples, ordered-collections

Should we add them in this PR?

Reply:
Yeah, you can add them in this PR.

Reply:
Done with d2393b51b443ca33db1cd6d683d46dcbbe3b8d0a :heavy_check_mark: